vault-core/src/main/java/org/apache/jackrabbit/vault/fs/api/package-info.java

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-@Version("2.7.0")
+@Version("2.8.0")
```

Comment:
why isn't this a patch increase?

Reply:
Don't really get that. This is a new feature and shouldn't be backported at all.